### ISSUE TITLE
How to restore drawings to TNZ with blank frames?

### ISSUE THREAD

**Issue by sharkfins13** [opentoonz/opentoonz#5428]
### Ask a question related to OpenToonz

Hi,
I am new to OpenToonz, don't really know any of the terminology, and am not sure how to go about fixing my scene/animation. I was working on my first animation the other day, as far as I am aware I saved all of the work that I did, but now when I open the file I can't see any of my drawings. I did save my TNZ to an external drive - is this the main issue and should I move it somewhere else?
It seems to have saved my 8 frames and 2 layers but I can't see the actual drawings on any of them, or the palette that I used. The text on each frame is red.
![empty frames](https://github.com/opentoonz/opentoonz/assets/168193817/4609511f-ce9a-4b0a-bc00-625c3ce7f029)
I'm not sure how to open the BAK file of the animation or even if it will have the drawings within it. I got the glut32.dll error several times as I tried opening different files but am not getting it any more.

Also on the external drive is one very pixelated PNG of one of my frames saved in the SceneIcons folder, which I guess is proof that something saved. I found these files (shown in image) within the Drawings section of my project, within the OpenToonz Stuff folder (:C), which I'm guessing is what I'm looking for??
![drawings files](https://github.com/opentoonz/opentoonz/assets/168193817/d3f22557-49d2-4db5-9f79-3cda7be48c8b)

Basically, can I/how do restore my drawings to my animation?
Thank you

**Comment by RodneyBaker** [comment]
The red lettering is an indication the link to where the images/level files are set to be are not in fact at that location.
If you Right Click on the level in the xsheet/timeline you should see Level Settings which will show you where Opentoonz expects the level to be.  This is normally \stuff\scenes\drawings (for TLV (toonz raster files).

If you have moved your stuff directory or content in the stuff directory then moving them back should allow Opentoonz to see them again.

Investigate that first as you might not need to resort to restoring the .Bak files if that resolves the problem.

Levels that are backed up are given the .bak extension.
Removing 'dot bak' will turn a file back into its original (recognizable) state (ex:  image.tlv.bak... becomes image.tlv.
In your screenshot .bak files aren't showing because Opentoonz doesn't recognize that .bak extension as a legitimate file extension.  Hence the need to remove for Opentoonz to see it again.
You might have to rename a newer level if your backup is the same name.  For instance if your level is 'A.tlv' and you attempt to remove the .bak extension from 'A.tlv.bak' you can't because a file already exists by that name.

**Comment by sharkfins13** [comment]
Your first solution worked fine, I just had to add the path of the drawings back to level settings, and my project is back to normal. I'm very relieved I didn't lose anything lol. Thanks :)
